modules/ContactsNitroModule/src/ContactsModule.nitro.ts

```diff
@@ -0,0 +1,29 @@
+import type { HybridObject } from 'react-native-nitro-modules'
+
+interface StringHolder {
```

Comment:
Is using `interface` a requirement of nitro modules? We are standardized on `type` everywhere else